### PR TITLE
Add UIDs to local dictionary and as a lint rule

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

File: content/en/docs/kubernetes/collector/components.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 linkTitle: Components
-spelling:
-  cSpell:ignore filelog crio containerd logtag gotime iostream varlogpods
-  cSpell:ignore varlibdockercontainers UIDs k8sattributes replicasets
-  cSpell:ignore kubeletstats kubelet cSpell:ignore replicationcontrollers
-  resourcequotas horizontalpodautoscalers cSpell:ignore statefulsets
+spelling: cSpell:ignore filelog crio containerd logtag gotime iostream kubelet
+spelling: cSpell:ignore varlibdockercontainers k8sattributes replicasets 
+spelling: cSpell:ignore kubeletstats resourcequotas horizontalpodautoscalers 
+spelling: cSpell:ignore replicationcontrollers statefulsets varlogpods
```
Comment (svrnm) [inline content/en/docs/kubernetes/collector/components.md:7]: the existing file is broken due to `prettier`, this way it will leave the block as is.

Comment (chalin) [reply content/en/docs/kubernetes/collector/components.md]: Oops, thanks for noticing.
I've submitted a proposed fix. PTAL

**Comment by chalin** [comment]
All checks are green, so I'll merge now. Feedback is still welcome @svrnm.

**Comment by svrnm** [comment]
thanks @chalin, lgtm

**Pull request merged**